src/plugins/lock/README.md

```diff
@@ -0,0 +1,15 @@
+- infos = Information about the lock plugin is in keys below
+- infos/author = Kurt Micheli <e1026558@student.tuwien.ac.at>
```

Comment:
Are you sure you want to use that e-mail address? I think they shut it down when you leave TU.